packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "0.6.3",
+  "version": "0.6.4",
```

Comment:
Have you used the "alpha package" workflow for the @labkey/components yet? I looked on artifactory and it looks like you haven't pushed this 0.6.4 version, which is good. The README for labkey-ui-components/package/components should have info (or ping myself, Susan, or Alan to walk you through it):
https://github.com/LabKey/labkey-ui-components/blob/master/packages/components/README.md

Reply:
👍